pkg/test/zq_example.go

```diff
@@ -0,0 +1,224 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/yuin/goldmark"
+	"github.com/yuin/goldmark/ast"
+	"github.com/yuin/goldmark/text"
+)
+
+type ZQExampleBlockType string
+
+const (
+	ZQ_COMMAND ZQExampleBlockType = "zq-command"
+	ZQ_OUTPUT  ZQExampleBlockType = "zq-output"
```

Comment:
The Go convention for names is MixedCaps (https://golang.org/doc/effective_go.html#mixed-caps).

Reply:
Done